@uportal/content-carousel/src/lib/RssStrategy.ts

```diff
@@ -37,14 +37,16 @@ export class RssStrategy implements DataStrategy {
             index: number,
         ) => {
           const image = media ? media.content : enclosures ? enclosures[0] : null;
+          const mediaImageUrl = media && Array.isArray(image) && image.length > 0 && image[0].$ ? image[0].$.url : image;
```

Comment:
This has enough complexity it may be worth breaking into it's own function, something like `extractImageFromRss`